packages/loader/driver-definitions/src/driverError.ts

```diff
@@ -97,13 +104,13 @@ export interface IAuthorizationError extends IDriverErrorBase {
 export interface IDriverBasicError extends IDriverErrorBase {
     readonly errorType:
     DriverErrorType.genericError
-    | DriverErrorType.authorizationError
```

Comment:
why remove this?

Reply:
authorizationError is part of more inclusive DriverError that is the type used across.
IDriverBasicError lists only types that use generic IDriverErrorBase, and authorizationError IAuthorizationError that is part of DriverError